lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -2073,7 +2073,7 @@ fn test_fail_htlc_on_broadcast_after_claim() {
 	let node_chanmgrs = create_node_chanmgrs(3, &node_cfgs, &[None, None, None]);
 	let mut nodes = create_network(3, &node_cfgs, &node_chanmgrs);
 
-	create_announced_chan_between_nodes(&nodes, 0, 1, InitFeatures::known(), InitFeatures::known());
+	let _ = create_announced_chan_between_nodes(&nodes, 0, 1, InitFeatures::known(), InitFeatures::known()).2;
```

Comment:
No need to change these lines at all now.

Reply:
Fixed